website/views/pages/try-fleet/sandbox-login.ejs

```diff
@@ -0,0 +1,44 @@
+<div id="sandbox-login" v-cloak>
+  <div style="max-width:  750px; padding-top: 80px; padding-bottom: 80px;" class="container d-flex flex-md-row flex-column justify-content-center align-items-center mx-auto px-4 px-md-3">
+    <div>
+    <img purpose="sandbox-image" style="width: 300px; height: auto;" src="/images/fleet-sandbox-300x244@2x.png" alt="Play in Fleet sandbox">
+    </div>
+    <div style="max-width: 400px;" class="flex-column d-flex">
+      <h2>Play in Fleet Sandbox</h2>
+      <p>
+        Fleet Sandbox is designed for testing Fleet features only.<br>
+        <a href="/docs/deploying">Click here</a> for production-ready deployments.
+      </p>
+      <div class="pt-4">
+        <ajax-form action="login" :syncing.sync="syncing" :cloud-error.sync="cloudError" :form-errors.sync="formErrors" :form-data="formData" :form-rules="formRules" @submitted="submittedLoginForm()" >
+          <div class="form-group mb-3">
+            <input id="emailAddress" type="text" class="form-control d-flex w-100" :class="[formErrors.emailAddress ? 'is-invalid' : '']" v-model.trim="formData.emailAddress" placeholder="Email">
+            <div class="invalid-feedback mt-2">This doesn’t appear to be a valid email address</div>
+          </div>
+          <div class="form-group mb-3">
+            <input id="password" type="password" class="form-control d-flex w-100" :class="[formErrors.password ? 'is-invalid' : '']" v-model.trim="formData.password" placeholder="Password">
+            <div class="invalid-feedback mt-2" v-if="formErrors.password === 'minLength'">Password too short.</div>
+
+            <div class="invalid-feedback mt-2" v-if="formErrors.password === 'custom'">Passwords must be at least 12 characters long, and must contain at least one letter, number and special character (!@#$%^&*).</div>
```

Comment:
I don't think you actually need anything other than "Please enter a password" here.  Like, if you enter a password that's too short, then it won't match.

Also if you leave out the validation here, then if password restrictions get gnarlier, people with existing passwords can still log in.

This would match what you're doing in the corresponding page script.  (i.e. neither of the "password is too short" or "password must be at least 12" messages would ever show, because neither validation is actually applied in the page script.)